package.json

```diff
@@ -27,8 +27,10 @@
     "test:watch": "mocha --watch",
     "mocks": "yarn run wiremock --verbose --port 8090 --root-dir test/mocks",
     "test:browser": "wait-on tcp:8090 tcp:5020 && API_BASE_URL=http://127.0.0.1:8090 cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js",
+    "test:browser1": "wait-on tcp:8090 tcp:5020 && API_BASE_URL=http://127.0.0.1:8090 cucumber-js test/browser/features/**/*.feature --require test/browser/support/env.js  --require test/browser/support/setup.js --require test/browser/step_definitions/**/*.js --tags @ukPass",
```

Comment:
is this needed?

Reply:
Hi Richa,
Yes. I need it to run tags independently whilst testing page by page, but also because we will be testing the happy path option by option. So please leave. If needs be I can demo locally for you.
Many Thanks
Sadie